app/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation "com.google.firebase:firebase-database:$rootProject.firebaseVersion"
     implementation "com.google.firebase:firebase-messaging:$rootProject.firebaseVersion"
     implementation "com.android.support.constraint:constraint-layout:$rootProject.constraintLayoutVersion"
+    implementation 'com.android.support.constraint:constraint-layout:1.1.0'
```

Comment:
You can remove this line as constraint layout is already added in the above line.